ios/app/MBXViewController.mm

```diff
@@ -186,7 +186,7 @@ - (void)parseFeaturesAddingCount:(NSUInteger)featuresCount
 
         if ([features isKindOfClass:[NSDictionary class]])
         {
-            NSMutableArray *annotations = [NSMutableArray array];
+            NSMutableArray <MBXAnnotation *> *annotations = [NSMutableArray array];
```

Comment:
Right, but that PR implements `MGLPointAnnotation` as the SDK-provided concrete implementation of `MGLAnnotation`; that's my point (heh). It mirrors MapKit. Devs will no longer have to create their own point annotation concrete implementation.